server/test/com/treode/server/ResourceSpec.scala

```diff
@@ -633,4 +633,178 @@ class ResourceSpec extends FreeSpec {
             .body (equalTo ("The slice must be between 0 (inclusive) and the number of slices (exclusive)."))
           .when
             .get ("/table/table1")
-        }}}}}
+        }}}}
+
+  "When post request contains batch-writes" - {
+
+    def addData (table: String, store: SchematicStubStore, key: String, value: String): TxClock =
+      update (store, TxClock.MinValue, key, value, table)
+
+    def updateData (table: String, store: SchematicStubStore, key: String, value: String): TxClock =
+      update (store, TxClock.now, key, value, table)
+
+    "and gives proper uris" - {
+
+      val body = "{\"v\":\"ant\"}"
+      val body2 = "{\"v\":\"ant2\"}"
+      val body3 = "{\"v\":\"ant3\"}"
+      val body4 = "{\"v\":\"ant4\"}"
+      val key = "\"abc\""
+      val key2 = "\"abc2\""
+      val key3 = "\"abc3\""
+      val key4 = "\"abc4\""
+
+      "POST /batch-write [{\"op\":\"CREATE\",\"table\":\"table1\",\"key\":\"abc\",\"obj\":{\"v\":\"ant\"}}] should yield Ok" in {
```

Comment:
Give these tests more friendly names; for example, `"POST /batch-write with one create"`.